tools/tflitefile_tool/select_operator.py

```diff
@@ -1253,6 +1254,28 @@ def GenerateModel(args, new_builder, sample_model, operator_list, new_input_tens
     return tflite.Model.ModelEnd(new_builder)
 
 
+def StoreMetadata(path, used_tensors, new_inputs, new_outputs):
```

Comment:
@ragmani Are you going to add more information in `metadata`?
At this moment, it contains only map original indices to new indices for input/output.
It would be helpful to use more descriptive name if you're going to add more properties in metadata.

Reply:
I have no plan to add mode information in `metadata` yet.
How about changing the name to `IOInfo`?